demisto_sdk/commands/common/hook_validations/playbook.py

```diff
@@ -1,19 +1,24 @@
+import os
 from typing import Dict
 
 import click
 from demisto_sdk.commands.common.errors import Errors
 from demisto_sdk.commands.common.hook_validations.content_entity_validator import \
     ContentEntityValidator
-from demisto_sdk.commands.common.tools import LOG_COLORS
+from demisto_sdk.commands.common.hook_validations.id import IDSetValidator
+from demisto_sdk.commands.common.tools import LOG_COLORS, open_id_set_file, print_warning
+from demisto_sdk.commands.create_id_set.create_id_set import IDSetCreator
 
 
 class PlaybookValidator(ContentEntityValidator):
     """PlaybookValidator is designed to validate the correctness of the file structure we enter to content repo."""
 
-    def is_valid_playbook(self, is_new_playbook: bool = True, validate_rn: bool = True) -> bool:
+    def is_valid_playbook(self, is_new_playbook: bool = True, validate_rn: bool = True,
+                          skip_dependencies: bool = False) -> bool:
         """Check whether the playbook is valid or not.
 
          Args:
+            skip_dependencies: whether dependencies should be skipped
```

Comment:
please expand this description, it currently doesn't add any more detailed data than the name of the argument